packages/cypress/src/support.ts

```diff
@@ -54,11 +54,27 @@ let gPluginServer: WebSocket | undefined;
 // user-facing
 const COMMAND_IGNORE_LIST = ["log-restore", "within-restore", "end-logGroup"];
 
+function handleReplayConnectResponse(v: unknown) {
+  if (v && typeof v === "object" && "port" in v && typeof v.port === "number") {
```

Comment:
purely a nit, but something like this is a bit cleaner for me

```js
if (typeof v?.port !== "number") {
  cy.log("[replay.io] Received unexpected response when connecting to plugin");
  return;
}
gServerPort = v.port;
```

Reply:
It is cleaner but it doesn't pass typechecking since `v` is unknown so more narrowing is required.